pkg/common/common.go

```diff
@@ -111,9 +111,12 @@ type Config struct {
 	// It is an optional function to customize model names.
 	GetDefinitionName func(name string) (string, spec.Extensions)
 
-	// PostProcessSpec runs after the spec is ready to serve. It allows a final modification to the spec before serving.
+	// PostProcessSpec runs after the V2 spec is ready to serve. It allows a final modification to the spec before serving.
 	PostProcessSpec func(*spec.Swagger) (*spec.Swagger, error)
 
+	// PostProcessSpec3 runs after the V3 spec is ready to serve. It allows a final modification to the spec before serving.
+	PostProcessSpec3 func(*spec3.OpenAPI) (*spec3.OpenAPI, error)
```

Comment:
Would you be able to put this in OpenAPIV3Config below and use it from there?

Reply:
that would make sense!  but....   but in apiserver, both the v2+v3 configs are `*openapicommon.Config`

https://github.com/kubernetes/apiserver/blob/master/pkg/server/config.go#L197
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/routes/openapi.go#L35

Reply:
That's a good reason to separate out the configs. Thanks for bringing this up.